pyspark/bigdl/nn/criterion.py

```diff
@@ -190,6 +190,27 @@ def __init__(self,
                                                     n_classes)
 
 
+class CosineDistanceCriterion(Criterion):
+
+    """
+    Creates a criterion that measures the loss given an input and target,
+    Loss = 1 - cos(x, y)
+
+
```

Comment:
No other parameters in this layer. Only size_average.

The x and y are the forward(x, y).